src/screen/DrawerScreen.tsx

```diff
@@ -178,6 +178,13 @@ function DrawerScreen({navigation}: DrawerContentComponentProps) {
             }
           />
           <Divider />
+          <ListItem
+            title="Push Settings"
```

Comment:
Since this is actually something the user will see, I think `Notifications` or `Notification Settings` would be better. What do you guys think? @ahoseinian, @geastwood, @stephanie-olp, @lmdobreva

Reply:
I added "Push"  so the user won't mixed it up with email notifications but as it is written in the text bellow "Notifications" should be enough

Reply:
done ✅